docs/ref/migration-operations.txt

```diff
@@ -204,6 +204,12 @@ default value, the operation is irreversible.
 
 .. _dependent objects outside the table are also dropped: https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-PARMS-CASCADE
 
+.. versionchanged:: 5.2
```

Comment:
We should remove the mention just above now I believe?

Reply:
Yeah, it was removed before but looks like it accidentally came back when adding the `versionchanged` annotation.

( https://github.com/django/django/pull/18209#discussion_r1618714748 )

Reply:
Yes, sorry for that, my bad. Fixed in latest commit.